src/CMakeLists.txt

```diff
@@ -42,10 +42,10 @@ endif()
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_STANDARD 20)
 
-# Hotfix for boost bug (https://github.com/boostorg/asio/issues/312):
-add_compile_options(-DBOOST_ASIO_DISABLE_CONCEPTS)
 SET(BOOST_MIN_VERSION "1.81.0")
-FIND_PACKAGE(Boost ${BOOST_MIN_VERSION} REQUIRED)
+# Old versions of CMake used the FindBoost module, which is deprecated because find_package can now find the upstream package directly.
```

Comment:
Not yours, but: lower case method call? And also limit the boost packages to "container date_time"?